renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -43,6 +43,7 @@ void BraveWalletRenderFrameObserver::DidCreateScriptContext(
   auto dynamic_params = get_dynamic_params_callback_.Run();
   if (!dynamic_params.brave_use_native_wallet) {
     js_ethereum_provider_.reset();
```

Comment:
shouldn't this call `WillReleaseScriptContext`? Or create another method called from both places?